<launch>
    <node name="pelican_ctrl" pkg="pelican_ctrl" type="PelicanCtrl" clear_params="true" output="screen" >
	      
	      <rosparam file="$(find pelican_ctrl)/launch/sim_param.yaml" />

<!--	        <remap to="vel_cmd" from="/fcu/control" />-->
            <remap from="pose" to="/fcu/gps_pose" />
          
 
    </node>
</launch>

